llama_index/llms/vllm.py

```diff
@@ -263,7 +263,8 @@ async def achat(
     async def acomplete(
         self, prompt: str, formatted: bool = False, **kwargs: Any
     ) -> CompletionResponse:
-        raise (ValueError("Not Implemented"))
+        kwargs = kwargs if kwargs else {}
+        return self.complete(prompt, **kwargs)
```

Comment:
does vllm support async call now?

Reply:
It doesn't support it for now but it has an internal asynchronous engine creation function which can be used to create an asynchronous generation function inside the LLM class. I am looking into and will try to create PR to create an agenerate function which will support asynchronous operation.

Reply:
But till that time, this temporary fix can work